src/ontology/cl-edit.owl

```diff
@@ -29211,7 +29212,8 @@ AnnotationAssertion(terms:contributor obo:CL_4023015 <https://orcid.org/0000-000
 AnnotationAssertion(oboInOwl:hasDbXref obo:CL_4023015 "ILX:0770150")
 AnnotationAssertion(oboInOwl:inSubset obo:CL_4023015 cl:BDS_subset)
 AnnotationAssertion(rdfs:label obo:CL_4023015 "sncg GABAergic cortical interneuron")
-EquivalentClasses(obo:CL_4023015 ObjectIntersectionOf(obo:CL_0000099 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015325)))
+EquivalentClasses(obo:CL_4023015 ObjectIntersectionOf(obo:CL_4023064 ObjectSomeValuesFrom(obo:RO_0002100 obo:UBERON_0000956) ObjectSomeValuesFrom(obo:RO_0002215 obo:GO_0061534) ObjectSomeValuesFrom(obo:RO_0002292 obo:PR_000015325)))
```

Comment:
Changing the parental terms produces that many cells cannot be classified as this cell type anymore.